docs/source/markdown/podman-buildfarm-create.1.md

```diff
@@ -0,0 +1,29 @@
+% podman-buildfarm-create 1
+
+## NAME
+podman\-buildfarm\-create - Create a new farm
+
+## SYNOPSIS
+**podman buildfarm create** [*options*] *name* [*connections*]
+
+## DESCRIPTION
+Create a new farm with connections that podman knows about that have been added via the
```

Comment:
Swich second "that have been" to "which were"

Reply:
fixed